tools/gethostlatency.bt

```diff
@@ -31,7 +31,10 @@
  * 08-Sep-2018	Brendan Gregg	Created this.
  */
 
-config = { missing_probes = "ignore" }
+config = {
+	missing_probes = "ignore";
+	unstable_macro = "enable";
```

Comment:
I'm a bit hesitant to use unstable features in these scripts. Do you mind just duplicating for now?

Reply:
Actually, I changed my mind. We're now using macros all over the place internally so it's probably fine.

Reply:
Yeah, I think it's good to use a bit but we shouldn't go too wild with them. The stdlib should be the primary playground until they are genuinely stable.